packages/admin/src/table/filterbar/FilterBar.tsx

```diff
@@ -0,0 +1,41 @@
+import { Theme } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import * as React from "react";
+
+import { FilterBarField, FilterBarFieldProps } from "./FilterBarField";
+import { FilterBarMoreFilters, FilterBarMoreFiltersProps } from "./FilterBarMoreFilters";
+
+export type CometAdminFilterBarClassKeys = "root" | "barWrapper" | "fieldBarWrapper";
+
+const useStyles = makeStyles(
+    (theme: Theme) => ({
+        root: {
+            "& [class*='CometAdminFormFieldContainer-root']": {
+                marginBottom: 0,
+            },
+        },
+        barWrapper: {
+            flexWrap: "wrap",
+            display: "flex",
+        },
+    }),
+    { name: "CometAdminFilterBar" },
+);
+
+interface FilterBarComposition {
```

Comment:
without the interface i cant declare FilterBar.MoreFilters = FilterBarMoreFilters
maybe i did something wrong but without the interface my eslint is marking it as an error

Reply:
Before looking into this issue, we have to decide on the naming: https://github.com/vivid-planet/comet-admin/pull/338#issuecomment-839531699

When using FilterBarMoreFilters this would be a non issue.